README.md

```diff
@@ -23,6 +23,7 @@ A collection of AMP tools making it easier to publish and host AMP pages. The fo
 - **[amp-optimizer](./optimizer):** a javascript library implementing server-side-rendering for AMP pages.
 - **[amp-optimizer-express](./optimizer-express)** an [express](http://expressjs.com/) middleware that applies AMP server-side-rendering on the fly.
 - **[amp-runtime-version](./runtime-version):** a javascript library for querying the current AMP runtime version.
+- **[amp-cache-url](./cache-url):** a javascript library for translating origin URLs to the AMP Cache URL format.
```

Comment:
Can you link AMP Cache URL format to https://developers.google.com/amp/cache/overview